src/features/shared-link-settings-modal/__tests__/SharedLinkSettingsModal-test.js

```diff
@@ -261,10 +277,18 @@ describe('features/shared-link-settings-modal/SharedLinkSettingsModal', () => {
             expect(section.prop('isDirectLinkUnavailableDueToDownloadSettings')).toEqual(
                 isDirectLinkUnavailableDueToDownloadSettings,
             );
+            expect(section.prop('isDirectLinkUnavailableDueToAccessPolicy')).toEqual(
+                isDirectLinkUnavailableDueToAccessPolicy,
+            );
             expect(section.prop('onChange')).toEqual(wrapper.instance().onAllowDownloadChange);
         });
     });
 
+    describe('renderModalTitle()', () => {
+        const wrapper = getWrapper();
+        expect(wrapper).toMatchSnapshot();
```

Comment:
Probably would be good to narrow the snapshot to just the title here since that's what this test is for, something like
```
expect(wrapper.find('.bdl-SharedLinkSettingsModal-title').toMatchSnapshot();
```

Reply:
You can also just shallow mount the response of renderModalTitle() and match that snapshot.